front/package-lock.json

```diff
@@ -5,7 +5,7 @@
   "packages": {
     "": {
       "dependencies": {
-        "@dust-tt/sparkle": "0.1.71-henry",
+        "@dust-tt/sparkle": "0.1.71",
```

Comment:
looks not rebased on main?

Reply:
It is actually, in previous PR I forgot to npm install after switching my sparkle (this is package lock)

Reply:
👍